datasetinsights/commands/train.py

```diff
@@ -105,6 +106,12 @@
     default=False,
     help="Force to disable validations.",
 )
+@click.option(
```

Comment:
It might be more convenient to have override string validation handled by click framework, similar to this:

https://github.com/Unity-Technologies/datasetinsights/blob/master/datasetinsights/commands/download.py#L12